tools-local/init-build.proj

```diff
@@ -130,8 +130,8 @@
       <PrebuiltPackages Include="$(PrebuiltPackagesPath)**/*.nupkg" />
       <PrebuiltSourceBuiltPackages Include="$(PrebuiltSourceBuiltPackagesPath)**/*.nupkg" />
     </ItemGroup>
-    <MarkAndCatalogPackages PackagesToMark="@(PrebuiltPackages)" CatalogOutputFilePath="$(PoisonReportDataFile)" MarkerFileName="$(PoisonMarkerFile)" />
-    <MarkAndCatalogPackages PackagesToMark="@(PrebuiltSourceBuiltPackages)" CatalogOutputFilePath="$(SourceBuiltPoisonReportDataFile)" MarkerFileName="$(SourceBuiltPoisonMarkerFile)" />
+    <MarkAndCatalogPackages PackagesToMark="@(PrebuiltPackages)" CatalogOutputFilePath="$(PoisonReportDataFile)" MarkerFileName="$(PoisonMarkerFile)" OverrideTempPath="$(LeakDetectionTempPath)" />
```

Comment:
Where is `LeakDetectionTempPath` set? I'm not seeing it in the repo.

Reply:
The VSO build machines actually have enough /tmp space without this.  I added it because the standard DevTestLab machines don't, so I had to set it to debug this issue.

Reply:
Ah, I expected the opposite usage, only in CI. Makes sense. :+1: